docs/API/LorisRESTAPI_v0.0.3.md

```diff
@@ -814,3 +814,49 @@ files, and a `.tar.gz` of the raw DICOM data as acquired during the candidate
 scanning session, and as retrieved from `/candidates/$CandID/$Visit/dicoms`.
 
 Only `GET` is currently supported.
+
+### 5.3 Upload DICOM file
+```
+PUT /candidates/$CandID/$VisitLabel/dicoms/$Filename/$IsPhantom
```

Comment:
I don't think $IsPhantom belongs in the URL, it would make more sense to add it to a request header and have something like

```
PUT /candidates/$CandID/$VisitLabel/dicoms/$Filename`
X-Is-Phantom: 1
```

or perhaps as a GET parameter

```
PUT /candidates/$CandID/$VisitLabel/dicoms/$Filename?phantom=1
```

But moreover, where is the file expected to live after being uploaded? If it's always going to be retrieved at 
` /candidates/$CandID/$VisitLabel/dicoms/$Filename` after being uploaded, this makes sense (but this section of the documentation should be combined with section 5.2, I think) but if the filename is derived from the header/patient data it would make more sense to have this a `POST` request to `/candidates/$CandID/$VisitLabel/dicoms` since the filename is discarded anyways.

Reply:
(Also, I think @xlecours's comment below is right and you can probably just derive whether or not it's a phantom from the CandID)

Reply:
The file is supposed to live in the location that is configured in Loris as given by 
MRI-Upload Directory which is specified Configuration->"Paths"

Reply:
That would be good to explicitly state in the documentation also, so that one knows where to look for uploaded files.

Reply:
Add it it as a request header is best I think so that the uploading the file happens only with the PUT request. 

So I will add that to the doc (along with an example) as well as update the current code to check for the X-Is-Phantom header.

Reply:
Regarding where the file is supposed to live, it will live in the location that is configured in Loris as given by MRI-Upload Directory which is specified Configuration->"Paths"

However I will state that explicitly in the documentation also so that one knows where to look for uploaded files.

Also adding IsPhantom as a request header in the PUT is best. This way uploading the DICOM fileset still happens only with the PUT request. Sticking IsPhantom in the url was feeling super kludgy but I couldn't think of something better at the time :-)

I will make the changes to the doc (along with giving an example) as well as update the current code to check for the X-Is-Phantom header.